drivers/npmx/npmx_shell.c

```diff
@@ -4190,20 +4143,19 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 	SHELL_CMD(soft_start, &sub_ldsw_soft_start, "LDSW soft-start", NULL),
 	SHELL_CMD(status, &sub_ldsw_status, "LDSW status", NULL), SHELL_SUBCMD_SET_END);
 
-/* Creating subcommands (level 3 command) array for command "leds mode". */
-SHELL_STATIC_SUBCMD_SET_CREATE(sub_leds_mode,
-			       SHELL_CMD(get, NULL, "Get LEDs mode", cmd_leds_mode_get),
-			       SHELL_CMD(set, NULL, "Set LEDs mode", cmd_leds_mode_set),
+/* Creating subcommands (level 3 command) array for command "led mode". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_led_mode, SHELL_CMD(set, NULL, "Set LED mode", cmd_led_mode_set),
+			       SHELL_CMD(get, NULL, "Get LED mode", cmd_led_mode_get),
 			       SHELL_SUBCMD_SET_END);
 
-/* Creating subcommands (level 3 command) array for command "leds state". */
-SHELL_STATIC_SUBCMD_SET_CREATE(sub_leds_state,
-			       SHELL_CMD(set, NULL, "Set LEDs state", cmd_leds_state_set),
+/* Creating subcommands (level 3 command) array for command "led status". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_led_status,
+			       SHELL_CMD(set, NULL, "Set LED status", cmd_led_status_set),
 			       SHELL_SUBCMD_SET_END);
 
-/* Creating subcommands (level 2 command) array for command "leds". */
-SHELL_STATIC_SUBCMD_SET_CREATE(sub_leds, SHELL_CMD(mode, &sub_leds_mode, "LEDs mode", NULL),
-			       SHELL_CMD(state, &sub_leds_state, "LEDs state", NULL),
+/* Creating subcommands (level 2 command) array for command "led". */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_led, SHELL_CMD(mode, &sub_led_mode, "LED mode", NULL),
+			       SHELL_CMD(status, &sub_led_status, "LED status", NULL),
```

Comment:
npmx function is called `npmx_led_state_set`

Reply:
Reverted to `state`.